src/utilities/gql.ts

```diff
@@ -1307,6 +1475,90 @@ const gql = {
     }
   `,
 
+  GET_EXTERNAL_EVENTS: `#graphql
+    query GetExternalEvents(
+      $sourceKey: String!,
+      $derivationGroupName: String!
+    ) {
+      ${Queries.EXTERNAL_EVENT}(
+        where: {
+          source_key: {_eq: $sourceKey},
+          derivation_group_name: {_eq: $derivationGroupName}
+        }
+      ) {
+        properties
+        event_type_name
+        key
+        duration
+        start_time
+        source_key
+      }
+    }
+  `,
+
+  GET_EXTERNAL_EVENT_BY_EVENT_TYPE: `#graphql
```

Comment:
Is this query still being used?

Reply:
Don't believe so - this is another older query that is no longer needed w. derivation groups. Removing